.github/workflows/build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   publish:
     name: Publish
     needs: build
-    if: github.event_name == 'push'
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push'
```

Comment:
C'est pourquoi ça?

Reply:
C'est pcq sinon chaque fois qu'on push dans une pr ca passait dans le step publish. Pcq le pipeline est triggered pour les pr et le event c'etait un push. Donc la en forçant un push sur master ca fait que ca roule le publish juste quand on merge dans master.